flink-filesystems/flink-s3-fs-hadoop/src/main/resources/META-INF/NOTICE

```diff
@@ -31,6 +31,7 @@ This project bundles the following dependencies under the Apache Software Licens
 - org.apache.httpcomponents:httpcore:4.4.6
 - org.apache.httpcomponents:httpclient:4.5.3
 - software.amazon.ion:ion-java:1.0.2
+- xerces:xercesImpl:2.12.0
```

Comment:
are we not also bundling `xml-apis:xml-apis`, a transitive dependency of xercesImpl?